portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsService.java

```diff
@@ -75,55 +88,36 @@ static VCard convertPersonToModel(Person person) throws IOException {
     Source: https://developers.google.com/people/api/rest/v1/people#personmetadata
     */
 
-    Pair<StructuredName, StructuredName[]> namesPair = convertToVCardNames(person.getNames());
-    vCard.setStructuredName(namesPair.first);
-    vCard.setStructuredNameAlt(namesPair.second);
-
-    /* TODO(olsona): uncomment when we want to test it
-    for (Telephone telephone : convertToVCardTelephoneNumbers(person.getPhoneNumbers())) {
-      vcard.addTelephoneNumber(telephone);
+    if (person.getNames() != null) {
+      Pair<StructuredName, StructuredName[]> namesPair = convertToVCardNames(person.getNames());
+      vCard.setStructuredName(namesPair.first);
+      vCard.setStructuredNameAlt(namesPair.second);
     }
 
-    for (ezvcard.property.Address vcardAddress : convertToVCardAddresses(person.getAddresses())) {
-      vcard.addAddress(vcardAddress);
+    if (person.getPhoneNumbers() != null) {
```

Comment:
maybe add a comment that the VCard API doesn't support adding multiple emails or telephones (if they did it would be a little cleaner)

Reply:
Done, and I also noticed that I could make things more efficient.

Reply:
nice